d3a_api_client/redis_client_base.py

```diff
@@ -48,29 +40,23 @@ def __init__(self, area_id, client_id, autoregister=True, redis_url='redis://loc
         if autoregister:
             self.register(is_blocking=True)
 
-    def _subscribe_to_response_channels(self, pubsub_thread=None):
-        channel_subs = {
-            self._response_topics[c]: self._generate_command_response_callback(c)
```

Comment:
Since we're deleting these lines, I think the `_generate_command_response_callback()` method can also be removed, right?
https://github.com/gridsingularity/d3a-api-client/blob/7fc1c0712c934022b16a6912891dab744e777b0b/d3a_api_client/redis_client_base.py#L107-L123

Reply:
You are totally right. Also in the redis market (we dont need the subscription to the responses of `"market_stats", "grid_fees", "dso_market_stats"` any more. Its handled in the batch commands. Thanks!